File: driver/lib/models/current_trip.dart
class CurrentTrip {
  final String id;
  final DateTime? actualTime;
  final DateTime intendedTime;
  final int capacityInVehicle;
  final int passengersQtyAfter;  
  final int passengersQtyBefore;  
  final int passengersQtyNew;  
  final String? stopId;
  final String? tripId;

  CurrentTrip ({
    required this.id,
    this.actualTime,
    required this.intendedTime,
    required this.capacityInVehicle,  
    required this.passengersQtyAfter, 
    required this.passengersQtyBefore,
    required this.passengersQtyNew,
    required this.stopId,
    required this.tripId,
  });

}

File: web/firebase/fireapp/models/currentTrip.py
from django.db import models

class CurrentTrip(models.Model):
    Id = models.CharField(max_length=200)
    ActualTime = models.DateTimeField()
    IntendedTime = models.DateTimeField()
    PassengersQtyAfter = models.IntegerField()
    PassengersQtyBefore = models.IntegerField()
    PassengersQtyNew = models.IntegerField()
    StopId = models.CharField(max_length=200)
    TripId = models.CharField(max_length=200)